tests/misc/tsizeof.nim

```diff
@@ -732,3 +732,11 @@ type
 
 doAssert sizeof(PackedUnion) == 11
 doAssert alignof(PackedUnion) == 1
+
+# bug #22553
+type
+  ChunkObj = object
+    data: UncheckedArray[byte]
+
+doAssert sizeof(ChunkObj) == 1
+doAssert offsetOf(ChunkObj, data) == 1
```

Comment:
Why? The offset of `data` should be 0.

Reply:
Because in https://github.com/nim-lang/Nim/pull/21979, it added a char dummy member prepended to objects only containing an `UncheckedArray`.
`c_offsetOf` also gives `1`.